geoapps/processing/peak_finder.py

```diff
@@ -2202,8 +2202,8 @@ def reset_groups(self, gates={}):
 
                 except IndexError:
                     print(
-                        "Could not find a time channel for the given list of time channels."
-                        + "Switching to non-tem mode"
+                        "Could not find a time channel for the given list of time channels.",
+                        "Switching to non-tem mode",
```

Comment:
**suggestion:** use string concatenation

here it uses multiple print parameters (still runtime, while string concatenation with `()` syntax is compile time).
```suggestion
                        ("Could not find a time channel for the given list of time channels. "
                        "Switching to non-tem mode")
```